flutter_todo/lib/screens/log_in.dart

```diff
@@ -36,8 +35,7 @@ class _LogInState extends State<LogIn> {
         _errorMessage = null;
       });
       try {
-        await app.logInUserEmailPw(
-            _emailController.text, _passwordController.text);
+        await app.logInUserEmailPw(_emailController.text, _passwordController.text);
```

Comment:
i notice a lot of stuff has been reformatted. i assume our editors have different auto-formatting configs. do you know how we can make these the same so we don't have this code changing all over the place?

Reply:
Yes we are working with width 500. If you want I can change it for this project.